src/Styles/Books.module.css

```diff
@@ -0,0 +1,7 @@
+.horizontalDivider {
+  height: 0.125rem;
+  background-color: #e8e8e8;
+  width: 100%;
+  margin: 2.5rem 0 1.813rem;
+  padding-bottom: 20px;
+}
```

Comment:
- [ ] As suggested by the previous reviewer, kindly consider adjusting the styles of the `border` above your `input` field and also the `input` field itself. The first `input` should cover more `width` as compared to that of the second. See below for more details:

<table>
<tr>
<th align=“middle”>Replin design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td><img width="1296" alt="Screenshot 2023-07-20 at 16 19 01" src="https://github.com/Kidd254/Bookstore-React-Redux/assets/107192076/6866eea1-f91e-4114-bd1d-2e4b4abf9535"></td>
<td>
<img width="1296" alt="Screenshot 2023-07-20 at 16 19 01" src="https://user-images.githubusercontent.com/104968236/254910622-c2f19d13-af77-48cc-9364-5d9ea74450b4.png">
</td>
</tr>
</table>